### PR TITLE
[ADD] Backported web_domain_field from 10.0 to 8.0

### PR DESCRIPTION
**Pull request by ssaid**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Travis RST errors unrelated

@lmignon what do you think of this?

File: web_domain_field/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Web Domain Field',
+    'summary': """
+        Use computed field as domain""",
+    'version': '8.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Eynes/E-Mips,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
```
Comment (MeetKD) [inline web_domain_field/__openerp__.py:12]: it should be https://github.com/OCA/web

File: web_domain_field/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'depends': [
+        'web'
+    ],
+    'data': [
+        'views/web_domain_field.xml',
+    ],
+    'demo': [
+    ],
```
Comment (MeetKD) [inline web_domain_field/__openerp__.py:20]: Can you please remove emtpy key ?

File: web_domain_field/static/src/js/pyeval.js [hunk 3]
```diff
@@ -0,0 +1,235 @@
+openerp.web_domain_field = function (instance){
+"use strict";
+
+// var pyeval = require('web.pyeval');
+var pyeval = instance.web.pyeval;
+// var session = require('web.session');
+var session = instance.session;
+
```
Comment (MeetKD) [inline web_domain_field/static/src/js/pyeval.js:8]: Can you please remove unused commented code ?

File: web_domain_field/__openerp__.py (hunk 1 above)
Comment (gabrielo77) [reply web_domain_field/__openerp__.py:12]: shouldn't it be both websites (Acsone and OCA) ?

**Comment by legalsylvain** [comment]
Hi @ssaid. thanks for this work ! 

One suggestion if possible. Could you cherry pick the old commit ? and commit your changes in the last commit. It makes the review easier to do.

kind regards.

**Pull request closed without merging**